### PR TITLE
[CodeSandbox] Examples: Advanced - Room Bot

### PR DESCRIPTION
**Pull request by sbis04**
Added the CodeSandbox of the **Room Bot** to the **Examples: Advanced** section.

Related issue: #796

### CODE REVIEW

File: docusaurus/docs/examples/advanced/room-bot.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+[![Powered by Wechaty](https://img.shields.io/badge/Powered%20By-Wechaty-brightgreen.svg)](https://github.com/Wechaty/wechaty)
+[![JavaScript](https://img.shields.io/badge/%3C%2F%3E-JavaScript-blue.svg)](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+
+## Try out the bot
+
+[![Edit wechaty-room-bot](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/sbis04/wechaty-room-bot/tree/main/?fontsize=12&hidenavigation=1&module=%2Froom-bot.js&theme=dark)
+
+You can try out the **Wechaty Room bot** using this interactive CodeSandbox.
+
+There is a `TODO` where you have to specify one of your friends' name (the CodeSandbox can be edited by using the above button). After that just scan the generated QR code with **WeChat** app, and you are ready to play with the bot!
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/docs/examples/advanced/room-bot.md]: > with **WeChat** app

I believe it will be better if we might use 'the' article before WeChat?

Comment (sbis04) [reply docusaurus/docs/examples/advanced/room-bot.md]: Nice catch! Have added it.

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Thanks for adding the rooms bot example docs page!

**Pull request merged**